### ISSUE TITLE
Dimension/coordinates issue with PS driver.

### ISSUE THREAD

**Issue by jtappin** [gnudatalanguage/gdl#1327]
There are definitely some issues with the PostScript driver and its handling of coordinates. While I've not got fully to the bottom of it, these two tests do seem to be shedding some light on the problem.

Test 1: Image position & size
````
img = bytscl(dist(200))

set_plot, 'ps'
device, /encap, xsize = 10., ysize = 10., file = 'test_cm.eps', $
        xoffset = 0., yoffset = 0., bits = 8

tv, img, .5, .5, xsize = 9., ysize = 9., /centi

device, /close

device, /encap, xsize = 10., ysize = 10., file = 'test_norm.eps', $
        xoffset = 0., yoffset = 0., bits = 8
tv, img, .05, .05, xsize = .9, ysize = .9, /norm
device, /close

end
````
I would expect the resulting files to look the same which they clearly do not:
![csys_image](https://user-images.githubusercontent.com/658624/167827707-fa8ecb61-9420-4600-9f64-83c39397339d.jpg)
The image on the left is using normalized coordinates (and looks correct), that on the right uses centimetres and doesn't.

Test 2: Isotropic plots.
````
th = findgen(361)*!dtor
x = cos(th)
y = sin(th)

set_plot, 'ps'
device, /encap, xsize = 10., ysize = 10., file = 'test_iso.eps'

plot, x, y, /iso
device, /close

end
````
This produces a plot which is very clearly not isotropic
![iso_plot](https://user-images.githubusercontent.com/658624/167828457-c9f33e73-1bd2-4ee0-a648-0d6f04ba922b.jpg)

Both do give the expected results in IDL. Also when no explicit page size is set, then the results look sensible.

My initial conjecture is that somewhere in the  coordinate conversion system a fixed page size is being assumed and the actual plot is then mapped onto the correct page size when the plot is generated.

**Comment by jtappin** [comment]
Just adding a little further data (I won't say clarification as I think it makes it more confusing).

1.  The various permutations of `CONVERT_COORD` do return the results expected given the actual plotting results.
2. `TV` in device coordinates does position the image correctly (i.e. replacing the centimetres example above with:
````
tv, img, .5*!d.x_px_cm, .5*!d.y_px_cm, xsize = 9.*!d.x_px_cm, $
    ysize = 9.*!d.y_px_cm, /dev
````
positions the image where wanted).

**Comment by GillesDuvert** [comment]
indeed it is a plplot postscript driver-only problem (svg does not have it).
As usual, since the PS driver use strange options and nightmarish coordinate transformations.
But...
There is hope since now gdl uses its own postscript driver...

**Comment by GillesDuvert** [comment]
solved in #1338
